lib/bootstrap_form/helpers/bootstrap.rb

```diff
@@ -59,6 +59,15 @@ def static_control(*args, &block)
         end
       end
 
+      def custom_control(*args, &block)
+        options = args.extract_options!
+        name = args.first
+
+        form_group_builder(name, options) do
+          capture(&block)
+        end
+      end
```

Comment:
Would this accomplish the same thing? Do we need the `extract_options!` and `capture`?

```ruby
def custom_control(name, options={}, &block)
   form_group_builder(name, options, &block)
end
```